packages/upswyng-web/src/components/ReportIssue.tsx

```diff
@@ -0,0 +1,187 @@
+    const selected: string[] = [];
+    Object.keys(issues).forEach(issue => {
+      if (issues[issue].checked) {
+        selected.push(issues[issue].text);
+      }
+    });
+    return JSON.stringify(selected);
+  };
+
+  const handleSubmit = () => {
+    // checking for empty stringified array
+    if (selectedIssues().length < 4 && !comments) {
+      setError(
+        "No feedback to submit. Please check desired boxes, enter comments, and try again."
+      );
+    } else {
+      axios
+        .post(`${serverUri}/api/resources/issues/user-report`, {
+          resourceId: resourceId,
+          detailExplanation: comments,
+          reportedIssues: selectedIssues(),
+        })
+        .then(() => setFormSubmitted(true))
+        .catch(() =>
+          setError(
+            "There was a problem sending your feedback. Please try again."
+          )
+        );
+    }
+  };
+
+  const handleCheck = (key: string) => (
+    event: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    updateIssues({
+      ...issues,
+      [key]: { ...issues[key], checked: event.target.checked },
+    });
+  };
+
+  if (resource === undefined) {
+    return <LoadingSpinner />;
+  }
+
+  return (
+    <Container>
+      <PageBanner
+        text="Report an Issue"
+        backButtonAction={lastLocation ? history.goBack : null}
+      />
+      <Typography component="div">Report an issue for: </Typography>
+      <Typography variant="h2" color="primary">
+        {resource?.name}
+      </Typography>
+      {formSubmitted ? (
+        <>
+          <Typography color="secondary" className={classes.marginTop}>
+            The problem has been submitted and will be reviewed. Thank you for
+            your feedback!
+          </Typography>
+          <Button
+            fullWidth
+            variant="contained"
+            color="primary"
+            onClick={history.goBack}
+            className={classes.marginTop}
+            size="medium"
+          >
+            Back
+          </Button>
+        </>
+      ) : (
+        <>
+          <FormGroup className={classes.marginTop}>
+            {Object.keys(issues).map((issue: string) => {
+              const { text, checked } = issues[issue];
+              return (
+                <FormControlLabel
+                  control={
+                    <Checkbox checked={checked} onChange={handleCheck(issue)} />
+                  }
+                  label={text}
+                  key={text}
+                />
+              );
+            })}
+          </FormGroup>
+          <TextField
+            multiline
+            fullWidth
+            className={classes.marginTop}
+            label="Comments"
+            variant="outlined"
+            value={comments}
+            onChange={e => setComments(e.target.value)}
+          />
+          <Button
+            fullWidth
+            variant="contained"
+            color="primary"
+            onClick={handleSubmit}
```

Comment:
Apologies for the delay @jollyjerr ! Could we make one last change and wrap all of these inputs in a `<form/>` and update this `handleSubmit` to be called by its `onSubmit` instead? After that I'm happy with this!